py/circuitpy_mpconfig.mk

```diff
@@ -39,38 +38,8 @@ ifndef CIRCUITPY_FULL_BUILD
 endif
 CFLAGS += -DCIRCUITPY_FULL_BUILD=$(CIRCUITPY_FULL_BUILD)
 
-# Setting CIRCUITPY_MINIMAL_BUILD = 1 will disable all features
-# Use for for early stage or highly restricted ports
-ifndef CIRCUITPY_MINIMAL_BUILD
-CIRCUITPY_MINIMAL_BUILD = 0
-endif
-CFLAGS += -DCIRCUITPY_MINIMAL_BUILD=$(CIRCUITPY_MINIMAL_BUILD)
-
-ifndef CIRCUITPY_DEFAULT_BUILD
-  ifeq ($(CIRCUITPY_MINIMAL_BUILD),1)
-        CIRCUITPY_FULL_BUILD = 0
-        CIRCUITPY_DEFAULT_BUILD = 0
-  else
-        CIRCUITPY_DEFAULT_BUILD = 1
-  endif
-endif
-CFLAGS += -DCIRCUITPY_DEFAULT_BUILD=$(CIRCUITPY_DEFAULT_BUILD)
-
-# Some features have no unique HAL component, and thus there's never
-# a reason to not include them.
-ifndef CIRCUITPY_ALWAYS_BUILD
-  CIRCUITPY_ALWAYS_BUILD = 1
-endif
-CFLAGS += -DCIRCUITPY_ALWAYS_BUILD=$(CIRCUITPY_ALWAYS_BUILD)
-
-
-# All builtin modules are listed below, with default values (0 for off, 1 for on)
-# Some are always on, some are always off, and some depend on CIRCUITPY_FULL_BUILD.
-#
-# *** You can override any of the defaults by defining them in your mpconfigboard.mk.
-
```

Comment:
Did you write this originally, or did Scott or I? I can't remember.

Reply:
It was me, right when I got started. As I mentioned in my reply to Sommersoft, I got stuck on thinking about the problem backwards and wanted ports to build up from things being disabled, rather than the other way around. Nobody else uses it so I think it's best deleted.